index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>David's Portfolio - Mobile</title>
+</head>
+<body>
+    <section class="works">
+        <div class="grid-container">
+            <div class="card card1">
+                <img class="screenshot" src="./pics/homepage.png" alt="">
+                
+                <h1 class="main-heading"> Tonic </h1>
+                
+                <ul class="top-container">
+                    <li>
+                        <p class="canopy">CANOPY</p>
+                    </li>
+                    <li><img src="./pics/dot.png" alt="dot"></li>
+                    <li>
+                        <p class="dev">Back End Dev</p>
+                    </li>
+                    <li><img src="./pics/dot.png" alt="dot"></li>
+                    <li>
+                        <p class="dev year">2015</p>
+                    </li>
+                </ul>
```

Comment:
- [ ] Please, consider for each this `ul` in your code the alignment to left. In your version, it is in the center.
>Your version
<img width="259" alt="Screen Shot 2022-02-11 at 18 12 06" src="https://user-images.githubusercontent.com/49064106/153607124-5e2d9f2e-d350-4501-90b0-7cc2817677bf.png">

> Figma design
<img width="353" alt="Screen Shot 2022-02-11 at 18 12 37" src="https://user-images.githubusercontent.com/49064106/153607176-908421e7-3ecd-4f30-b122-c3da95652e20.png">

Reply:
Well noted. I have edited the Ul element and pushed it to the left.